pkg/network/apis/network/validation/validation.go

```diff
@@ -56,6 +82,9 @@ func ValidateClusterNetwork(clusterNet *networkapi.ClusterNetwork) field.ErrorLi
 		if clusterNet.HostSubnetLength != defaultClusterNetwork.HostSubnetLength {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("hostSubnetLength"), clusterNet.HostSubnetLength, "cannot change the default ClusterNetwork record via API."))
 		}
+		if !reflect.DeepEqual(clusterNet.ClusterNetworks, defaultClusterNetwork.ClusterNetworks) {
```

Comment:
cluster networks could be same but the order in the list could be different?

Reply:
A change in the order of the cluster network at this point changes which chunk subnet allocators are created, so I thought it would be better to have it consistant and only change when master is restarted